plugins/types.ts

```
@@ -0,0 +1,58 @@
+    context: T
+  ) => void | Promise<void>;
+  after?: (context: T, bunextRequest: BunextRequest) => void | Promise<void>;
+};
+
+export type AfterBuildMain = () => Promise<any> | any;
+export type BeforeBuild = () => Promise<any> | any;
+
+export type Request_Plugin = (
+  request: BunextRequest
+) =>
+  | Promise<void | undefined | BunextRequest>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)